dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -251,10 +260,6 @@ namespace dwa_local_planner {
 
   bool DWAPlannerROS::computeVelocityCommands(geometry_msgs::Twist& cmd_vel) {
     // dispatches to either dwa sampling control or stop and rotate control, depending on whether we have been close enough to goal
-    if ( ! costmap_ros_->getRobotPose(current_pose_)) {
-      ROS_ERROR("Could not get robot pose");
-      return false;
-    }
```

Comment:
why is this removed?

Reply:
It's explained in line 157 of my dwa_planner_ros.cpp.
If we update robot pose after setting time in planner_util we can get a more recent time stamp of robot pose, which will cause TF 'extrapolation into the future` error in line
https://github.com/ros-planning/navigation/blob/43e9297d588276efa9868efe2ed56ac6bf08e180/base_local_planner/src/goal_functions.cpp#L115